docker-compose.oidc.yaml

```diff
@@ -0,0 +1,86 @@
+version: "3.8"
+
+services:
+  mysql:
+    image: mysql:8.0
+    user: "${MYSQL_UIDGID:-mysql}"
+    restart: on-failure
+    environment:
+      MYSQL_DATABASE: "${MYSQL_DATABASE}"
+      MYSQL_USER: "${MYSQL_USER}"
+      MYSQL_PASSWORD: "${MYSQL_PASSWORD}"
+      MYSQL_ROOT_PASSWORD: "${MYSQL_ROOT_PASSWORD}"
+      KEYCLOAK_DATABASE: "${KEYCLOAK_DATABASE}"
+      KEYCLOAK_DB_USER: "${KEYCLOAK_DB_USER}"
+      KEYCLOAK_DB_PASSWORD: "${KEYCLOAK_DB_PASSWORD}"
+    ports:
+      - "${MYSQL_HOST_PORT}:3306"
+    volumes:
+      - ${MYSQL_VOLUME}:/var/lib/mysql
+      - ./init:/docker-entrypoint-initdb.d
+  minio:
+    image: minio/minio:RELEASE.2021-02-11T08-23-43Z
+    user: "${MINIO_UIDGID:-}"
+    restart: on-failure
+    environment:
+      MINIO_ACCESS_KEY: "${MINIO_ACCESS_KEY}"
+      MINIO_SECRET_KEY: "${MINIO_SECRET_KEY}"
+    ports:
+      - "${MINIO_HOST_PORT}:9000"
+    volumes:
+      - ${MINIO_VOLUME}:/data
+    command: server /data
+  app:
+    build: flask
+    image: ghcr.io/ccmbioinfo/stager:dev
+    entrypoint:
+      [
+        "./utils/wait-for-it.sh",
+        "mysql:3306",
+        "--timeout=0",
+        "--",
+        "./utils/wait-for-it.sh",
+        "keycloak:8080",
+        "--timeout=0",
+        "--",
+        "./utils/run.sh",
+      ]
+    command: ["--host=0.0.0.0"]
+    user: "${FLASK_UIDGID:-www-data}"
+    environment:
+      ST_SECRET_KEY: "${ST_SECRET_KEY}"
+      ST_DATABASE_URI: "mysql+pymysql://${MYSQL_USER}:${MYSQL_PASSWORD}@mysql/${MYSQL_DATABASE}"
+      MINIO_ENDPOINT: minio:9000
+      MINIO_ACCESS_KEY: "${MINIO_ACCESS_KEY}"
+      MINIO_SECRET_KEY: "${MINIO_SECRET_KEY}"
+      ENABLE_OIDC: "true"
+      OIDC_CLIENT_ID: "ccm-stager"
+      OIDC_CLIENT_SECRET: "$$not_a_secret"
+      OIDC_WELL_KNOWN: "http://keycloak:8080/auth/realms/ccm/.well-known/openid-configuration"
+      OIDC_PROVIDER: "keycloak"
+      KEYCLOAK_HOST: "http://keycloak:8080"
```

Comment:
Might need to add KEYCLOAK_USER and PW here, as they are referenced in `manage_oidc.py` (I needed to add them manually to use that function outside of the initialization code).